lib/server_health_check_rails/logger.rb

```diff
@@ -0,0 +1,63 @@
+# The health_index_path route is pinged frequently. We don't want that to clutter
+# our main log file, so we intercept all calls to that route and change the logger.
+module ServerHealthCheckRails
```

Comment:
See my comment on `app/models/server_health_check_rails/health_check.rb`. I think this file should be moved to your specific application.